schema.graphql

```diff
@@ -1,21 +1,25 @@
 type Transfer @jsonField {
-  extrinsicHash: String
   amount: String!
   to: String!
   from: String!
   fee: String!
-  block: String!
-  extrinsicId: String
+  eventIdx: Int!
```

Comment:
What index to we put for failed extrinsic? Maybe consider optional

Reply:
I agree with you, but in generated code the property is not optional
<img width="705" alt="Screenshot 2021-08-31 at 14 41 50" src="https://user-images.githubusercontent.com/17689921/131496510-28ec5b19-60b3-466b-a445-fea0f5df0847.png">